examples/PresignedGetObject.java

```diff
@@ -40,8 +41,9 @@ public static void main(String[] args)
       // Get presigned URL string to download 'my-objectname' in 'my-bucketname' and its life time
       // is one day.
       String url =
-          minioClient.presignedGetObject(
-              PresignedGetObjectArgs.builder()
+          minioClient.getPresignedObjectUrl(
+              GetPresignedObjectUrlArgs.builder()
+                  .method(Method.GET)
```

Comment:
Shall the two files `PresignedGetObject.java` and `PresignedPutObject.java` be merged into a single example file called `GetPresignedObjectUrl.java` ?